sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1203,6 +1238,10 @@ void merge(final @NotNull SentryOptions options) {
     if (options.getProxy() != null) {
       setProxy(options.getProxy());
     }
+    final Map<String, String> tags = new HashMap<>(options.getTags());
+    for (Map.Entry<String, String> tag : tags.entrySet()) {
+      this.tags.put(tag.getKey(), tag.getValue());
```

Comment:
should we define precedence here? if both have the same `keys`, which one should win?

Reply:
The one that is passed as a parameter. External configuration overwrites the configuration provided in the code. This precedence is verified in unit tests.